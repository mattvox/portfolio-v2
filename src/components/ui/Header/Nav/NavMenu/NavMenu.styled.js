import Link from 'gatsby-link';
import styled from 'styled-components';

export const MenuWrapper = styled.div`
  flex: 1 0 100%;
`;

export const NavLink = styled(Link)`
  display: block;
  padding: 1em;
  text-decoration: none;
  font-size: 1.2rem;
`;
