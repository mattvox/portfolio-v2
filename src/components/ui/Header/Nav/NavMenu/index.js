import React from 'react';

import { MenuWrapper, NavLink } from './NavMenu.styled';

const NavMenu = () => (
  <MenuWrapper>
    <NavLink to="/">Home</NavLink>
    <NavLink to="/about">About</NavLink>
    <NavLink to="/">Resume</NavLink>
    <NavLink to="/">Contact</NavLink>
  </MenuWrapper>
);

export default NavMenu;
