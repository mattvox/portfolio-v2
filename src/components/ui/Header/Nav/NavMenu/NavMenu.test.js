import React from 'react';

import NavMenu from '../NavMenu';

describe('NavMenu', () => {
  it('should render correctly', () => {
    const output = shallow(<NavMenu />);

    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
