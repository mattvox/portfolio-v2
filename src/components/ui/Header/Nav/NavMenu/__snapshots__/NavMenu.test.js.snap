// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavMenu should render correctly 1`] = `
<styled.div>
  <Styled(GatsbyLink)
    to="/"
  >
    Home
  </Styled(GatsbyLink)>
  <Styled(GatsbyLink)
    to="/about"
  >
    About
  </Styled(GatsbyLink)>
  <Styled(GatsbyLink)
    to="/"
  >
    Resume
  </Styled(GatsbyLink)>
  <Styled(GatsbyLink)
    to="/"
  >
    Contact
  </Styled(GatsbyLink)>
</styled.div>
`;
